base/subarray.jl

```diff
@@ -280,6 +280,13 @@ AbstractZeroDimArray{T} = AbstractArray{T, 0}
 
 reindex(::Tuple{}, ::Tuple{}) = ()
 
+_maybeview(A, i1::Integer, inds::Integer...) = (@_propagate_inbounds_meta; A[i1, inds...])
+function _maybeview(A, v...)
```

Comment:
Why/how is this different from Base.maybeview? Maybe just use that instead?

Reply:
It's not, I had forgotten about it. Had a nagging feeling that I was missing something. Updated now.